contracts/proxy/library.cairo

```diff
@@ -0,0 +1,64 @@
+%lang starknet
+
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+from starkware.starknet.common.syscalls import get_caller_address
+
+#
+# Storage variables
+#
+
+@storage_var
+func Proxy_implementation_address() -> (implementation_address: felt):
+end
+
+@storage_var
+func Proxy_admin() -> (proxy_admin: felt):
+end
```

Comment:
It might be a good idea to include Proxy getters for these variables—especially for testing